### ISSUE TITLE
Installation-Docs are broken. Not working on Ubuntu or Centos... 

### ISSUE THREAD

**Issue by Strange-Panda** [credativ/elephant-shed#23]
I would like to inform you that the installation instructions for Elephant-shed on Ubuntu and Centos do not work.

We have now tried seven times with Ubuntu 20.04 and three times with 18.04. Without success. The installation of Ubuntu was fresh each time and it followed their guide 1:1.

Grafana has no dashboard when invoked.
Also, OmniDB sometimes doesn't work at all, sometimes it doesn't accept the password .

Are these errors known? Is there possibly a work-around?

On Centos we can not even load the repository. Credativ is not responding for E-Mails. 

> yum install https://packages.credativ.com/public/postgresql/yum/credativ-repo.rpm
> Elephant Shed for Red Hat/CentOS 8 - x86_64                                                                                                                           3.4 kB/s | 268  B     00:00
> Errors during downloading metadata for repository 'credativ-arch':
>   - Status code: 404 for https://packages.credativ.com/public/postgresql/yum/el8/x86_64/repodata/repomd.xml (IP: 93.94.130.77)
> Fehler: Failed to download metadata for repo 'credativ-arch': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried

**Comment by Strange-Panda** [comment]
Debian 9: 

> 
> credativ
> Sorry, a 404 error occurred while loading https://10.100.110.162/omnidb/.
> Entschuldigung, beim Zugriff auf https://10.100.110.162/omnidb/ trat ein 404-Fehler auf.
> Elephant Shed -- report bugs on GitHub
> 

And again: No Dashboard in Grafana. 

Everything seems to be wrong with this Elephant-Shed

**Comment by heralight** [comment]
About Omnidb problem a fix that work on ubuntu 20.04:

Hello credativ Team.
I am facing the same issue as mentioned above. I have a clean install on ubuntu 20.04 and i cannot login to OmniDB. I'm trying to login with user `admin` and password `admin` but it says that the username or password is invalid. I did the following to make it work:
```bash
touch $HOME/.omnidb/omnidb-server/config.py
omnidb-server -C $HOME/.omnidb/omnidb-server/config.py -r
sudo systemctl restart omnidb-server.service
```

_Originally posted by @strapsi in https://github.com/credativ/elephant-shed/issues/21#issuecomment-838351993_

**Comment by heralight** [comment]
And for Grafana, the dashboard is under `Dasboards->Manage->PostgreSQL Server Overview`

**Comment by Strange-Panda** [comment]
I found the Grafana Dashboard. Thank you for mentioning. 
But the Dashboard shows no Data. 
If i look in prometheus, i can see that two endpoints are down: 

http://localhost:9100/metrics 
http://localhost:9237/metrics

admin/admin on OmniDB is still a problem.

**Comment by mbanck** [comment]
> Debian 9:
> 
> > credativ
> > Sorry, a 404 error occurred while loading https://10.100.110.162/omnidb/.
> > Entschuldigung, beim Zugriff auf https://10.100.110.162/omnidb/ trat ein 404-Fehler auf.
> > Elephant Shed -- report bugs on GitHub

I'm afraid omnidb is not supported on stretch, there are no stretch packages for OmniDB we are ware of.

**Comment by df7cb** [comment]
The dashboard needs input from prometheus-sql-exporter, so if it doesn't work, try restarting prometheus-sql-exporter.service, waiting a bit, and reload the grafana page. I just fixed some issues around booting that should be resolved in the upcoming 1.3.5 release.
